servor.js

```diff
@@ -132,6 +139,6 @@ module.exports = async ({
     root,
     protocol,
     port,
-    ips
+    ips,
```

Comment:
@lukejacksonn do we need this dangling comma?

Reply:
perhaps this is an editor config? there are a lot of dangling commas in the code now. 🙈

Reply:
Yeh I realised.. this is generally my personal preference (for browser based projects at least).. when was this supported in node do you know? Is it likely to cause issues?

Reply:
I've just reverted that change.. it didn't belong in this PR 😅 now the `servor.js` diff is all relevant to the fix!